portal-web/test/functional/com/liferay/portalweb/paths/portlet/remoteapps/RemoteAppsEntry.path

```diff
@@ -30,6 +30,16 @@
 	<td>//textarea[contains(@id,'properties')]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>ADD_URL_ROW</td>
+	<td>//*[contains(@id,'${key_id}')]//button[contains(@class,'add-row')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>URL_ROW</td>
```

Comment:
Was debating a rename between `URL_ROW_INDEX` or `URL_INDEX`, but either is fine with me. Alphabetize so it's directly below URL path and more easily findable.